submissions/NwagboJohn/HtmlTask/style.css

```diff
@@ -0,0 +1,29 @@
+
+	<style type="text/css">

```

Comment:
you need not put a `style` tag in your css file. this tag is only needed when you write css within html. remove the tag at the beginning and end of the file and format appropriately.

see https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md